highfive/newpr.py

```diff
@@ -62,7 +62,26 @@ def review_msg(reviewer, submitter):
         text = review_with_reviewer % reviewer
     return text
 
-reviewer_re = re.compile("\\b[rR]\?[:\- ]*@([a-zA-Z0-9\-]+)")
+# This regular expression has a lot going on. Here are explanations to go along
+# with the numbers below.
+#
+# 1. The expression is either at the beginning of a string or is preceded by
+#    whitespace.
+# 2. The target string begins with 'r' or 'R' followed by a '?'.
+# 3. The '?' is followed by up to one ':' or '-'.
+# 4. At least one whitespace character separates the first pattern from the
```

Comment:
Should it be exactly one space rather than any nonzero amount of whitespace?

Reply:
Again, lets go for more flexible